frontend/pages/artist/_itemId/index.vue

```diff
@@ -152,6 +269,10 @@ export default Vue.extend({
     return isValidMD5(ctx.route.params.itemId);
   },
   async asyncData({ params, $api }) {
+    const albumBreakpoints = {
+      singleTicksMaxLength: 6000000000,
+      epTicksMaxLength: 18000000000
+    };
```

Comment:
Can we store these as ms and convert between both using [our helper](https://github.com/jellyfin/jellyfin-vue/blob/master/frontend/utils/time.ts#L14)?

Ticks is specific to .NET world and we talked some times about moving to seconds in the v2 API, besides is user friendlier for new developers to find most code references to milliseconds instead of "ticks".

Reply:
I'll switch it over.

Reply:
I've switched to using ms instead of ticks and converted for comparison.